doc/security/specs/identities_and_root_keys/README.md

```diff
@@ -121,78 +124,71 @@ Hidden from software once personalization is complete.
-2. Use a software binding tag stored in the ROM_EXT manifest. This is to
-   retain the Creator Identity across validated updates of the ROM_EXT.
-   The implementation may opt to use the software binding interface
-   described in later sections to fulfill this property.
+SHA-3-256 hash of the ROM image.
     </td>
   </tr>
   <tr>
-    <td>DeviceIdentifier</td>
-    <td>OTP</td>
+    <td>Health State Measurement</td>
+    <td>Computed by Lifecycle controller</td>
     <td>
-Provisioned at manufacturing time. Readable from software and JTAG interface.
+Comprises the following measurements:
+
+* Device life cycle state.
+* Debug mode state.
+
+The debug mode shall be used as well if there are multiple debug configurations
+supported by a single life cycle state.
     </td>
   </tr>
   <tr>
-    <td>HardwareRevisionSecret</td>
-    <td>Gates</td>
+    <td>DeviceIdentifier</td>
+    <td>OTP</td>
     <td>
-Encoded in gates. Provisioned by Silicon Creator before tapeout. Hidden from
-software.
+Provisioned at manufacturing time. Readable from software and JTAG interface.
     </td>
   </tr>
   <tr>
-    <td>Health State Measurement</td>
-    <td>Register (ROM stage)</td>
+    <td>SoftwareBindingValue</td>
+    <td>SW register</td>
     <td>
-Comprises the following measurements:
-
-* Device life cycle state.
-* Debug mode state.
-* ROM Hash.
+For the attestation chain, this register is populated with the hash of the OTP
+image. Therefore, changes in the OTP configuration will trigger an update of
+the CreatorIdentity for attestation.
 
-Some values are read from the device life cycle controller. The device life
-cycle state should be consumed by the ROM stage.
-
-The debug mode shall be used as well if there are multiple debug configurations
-supported by a single life cycle state.
+For the sealing chain, this register uses a binding tag stored in the ROM_EXT
+manifest. This is to retain the same CreatorIdentity across validated updates of
+the ROM_EXT. This allows sealing keys to be persistent across ROM_EXT updates.
     </td>
   </tr>
 </table>
 
-The CreatorRootKey can be used to generate the Creator Identity key and the
+The CreatorRootKey can be used to generate the CreatorIdentity and the
 OwnerIntermediateKey described in the following sections.
 
-## Creator Identity
+## CreatorIdentity
 
-The Creator Identity is an asymmetric key derived from the `CreatorRootKey`. It
+The CreatorIdentity is an asymmetric key derived from the `CreatorRootKey`. It
 is used as a cryptographic identifier bound to the device and the Silicon
 Creator. It is used to attest to the authenticity of the physical device and the
-ROM and ROM\_EXT configuration.
+ROM and OTP configuration.
 
-The Creator Identity is generated as follows:
+CreatorIdentity becomes part of the UDS certificate and it is
+generated as follows:
 
 ```
 CreatorIdentitySeed =
-  KM_DERIVE(CreatorRootKey, IdentityDiversificationConstant)
+  KM_DERIVE(CreatorRootKey, OTBNDiversificationConstant)
```

Comment:
Not sure what you are referring to by the `OTBNDiversificationConstant`, but if it is this the [diversification constant used by the output-generate function for CreatorIdentitySeed](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/lib/cert/dice.c;l=78?q=sc_keymgr_diversification_t&ss=opentitan%2Fopentitan), then I would not say it is specific to OTBN, it is specific to the output-generate invocation we are using for this specific UDS (CreatorIdentity) keygen, i.e., it is the `CreatorIdentityDiversifier` or something like that.

Reply:
If you are [referring to this](https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/keymgr/rtl/keymgr.sv;l=30?q=otbn&ss=opentitan%2Fopentitan:hw%2Fip%2Fkeymgr%2F), then I would say that the KM_DERIVE needs an additional input then as well (the `CreatorIdentityDiversifier` I mentioned as well).

Reply:
Added `CreatorIdentityDiversifier` as an extra input.